server/controllers/konnectors.coffee

```diff
@@ -77,6 +50,24 @@ module.exports =
 
             res.status(204).send konnector
 
+    update: (req, res, next) ->
+        # Don't update connector if an import is already running.
+        if req.konnector.isImporting
+            res.send 400, message: 'konnector is importing'
```

Comment:
You're right. Let's keep it as it for the moment and let's just keep in mind that we should think about those "already importing" status. I like your propositions 👍